src/electron/device-disconnected-popup/device-disconnected-popup.scss

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../common/styles/fonts.scss';
+
+.title-container {
```

Comment:
the name sounds generic. title-container, ms-Dialog-header & title-text.

Either wrap this under a parent container called say, device-disconnected-container or make the names more specific or move them to a common scss file if you want them to be common for other titles & headers too.

Reply:
I did 2 things:
- Add a class name for the dialog so we can "group" this class names inside it
- Enabled CSS modules so the final class names are made unique (on prod at least) by the build process.

As per `ms-Dialog-header`, that comes from office fabric, we have no way to directly change it

Reply:
Not sure if the change has been made in master but it seems like electron always uses dev config instead of the prod config.

For now that means the 2nd item you mentioned isn't true _yet_. Just something to note.

Reply:
That's correct... we don't have a prod config for electron. But I saw the config is there for us to use when the time comes.